src/test/java/jalau/at18/katas/likes/fernanda/MainTest.java

```diff
@@ -0,0 +1,16 @@
+package jalau.at18.katas.likes.fernanda;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public class MainTest {
+
+    @Test
+    public void shouldReturnHighestLowest() {
+        String input = "Max,John,Mark";
+        String expected = "Max , John and Mark Like this";
```

Comment:
There is a unexpected space between "Max" and ",", this does not match with kata requirement
```
["Max", "John", "Mark"]           -->  "Max, John and Mark like this"
```
Please update it so it matches kata requirement.

Also, please add tests or assertions that also verify that other kata requirements are also validated
```
[]                                -->  "no one likes this"
["Peter"]                         -->  "Peter likes this"
["Jacob", "Alex"]                 -->  "Jacob and Alex like this"
["Alex", "Jacob", "Mark", "Max"]  -->  "Alex, Jacob and 2 others like this"
```

Reply:
The unexpected space was removed and more tests were added